src/driver/amdxdna/aie2_smu.c

```diff
@@ -134,6 +159,29 @@ int aie2_smu_set_dpm_level(struct amdxdna_dev_hdl *ndev, u32 dpm_level)
 	if (!ret)
 		XDNA_INFO_ONCE(ndev->xdna, "Set soft dpm level = %d", dpm_level);
 
+	ndev->smu_curr_dpm_level = dpm_level;
+
+	return ret;
+}
+
+int aie2_smu_get_dpm_level(struct amdxdna_dev_hdl *ndev)
+{
+	return ndev->smu_curr_dpm_level;
+}
+
+int aie2_smu_set_dpm_level(struct amdxdna_dev_hdl *ndev, u32 dpm_level)
+{
+	int ret;
+
+	if (SMU_DPM_MAX(ndev) == 0 || dpm_level > SMU_DPM_MAX(ndev))
+		return 0;
+
+	if (!ndev->priv->smu_rev) {
+		ret = aie2_smu_set_dpm_level_v0(ndev, dpm_level);
+	} else {
+		ret = aie2_smu_set_dpm_level_v1(ndev, dpm_level);
+	}
```

Comment:
```suggestion
	if (!ndev->priv->smu_rev)
		ret = aie2_smu_set_dpm_level_v0(ndev, dpm_level);
	else
		ret = aie2_smu_set_dpm_level_v1(ndev, dpm_level);
```

See https://www.kernel.org/doc/html/v4.10/process/coding-style.html search "Do not unnecessarily use braces where a single statement will do."

Reply:
Done